src/models/chen2021Fitness/Chen2021FitnessWidget.tsx

```diff
@@ -0,0 +1,381 @@
+import { Widget } from '../../widgets/Widget';
+import { ZodQueryEncoder } from '../../helpers/query-encoder';
+import { SampleSelectorSchema } from '../../helpers/sample-selector';
+import * as zod from 'zod';
+import React, { useEffect, useState } from 'react';
+import { get } from '../../services/api';
+import {
+  Chen2021FitnessRequestSchema,
+  Chen2021FitnessResponseSchema,
+  ValueWithCISchema,
+} from './chen2021Fitness-types';
+import styled from 'styled-components';
+import { Col, Form, Button } from 'react-bootstrap';
+import { Plot } from '../../components/Plot';
+
+const ContainerPropsSchema = SampleSelectorSchema;
+type ContainerProps = zod.infer<typeof SampleSelectorSchema>;
+
+const ResultsPropsSchema = zod.object({
+  request: Chen2021FitnessRequestSchema,
+});
+type ResultsProps = zod.infer<typeof ResultsPropsSchema>;
+
+type Chen2021FitnessRequest = zod.infer<typeof Chen2021FitnessRequestSchema>;
```

Comment:
`ResultsProps` and `Chen2021FitnessRequest` should not use Zod. It is important for API _responses_ or props loaded _from a URL_ (data we get from the server or user), since invalid data will cause crashes and weird bugs, even though the TS types say they shouldn't happen. For requests it just complicates things and makes TS print less nice errors.

Note that it's still required to use Zod for `ContainerProps`, since that is used to check the props when they get loaded from the URL.